src/org/scada_lts/web/security/XssUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.scada_lts.web.security;
+
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+import java.util.List;
+import java.util.Arrays;
+
+public class XssUtils {
+
+    private XssUtils() {
+        throw new UnsupportedOperationException("Utility class");
```

Comment:
This is a no good approach, we will find out in runtime. 
It's better to make a final class to highlight it in the IDE and create a private no args constructor.